SDSetupBackend/Controllers/v1.cs

```diff
@@ -86,7 +86,7 @@ public class v1 : ControllerBase {
                     DeletingFileStream stream = (DeletingFileStream) ZipFromFilestreams(files.ToArray(), uuid);
 
                     Program.generatedZips[uuid] = stream;
-                    stream.Timeout(30000);
+                    stream.Timeout(180000);
```

Comment:
I don't want zips sitting on the server for 3 minutes if they aren't going to be downloaded.

Reply:
That's fine.  The problem I was seeing and that this fixed, is that zips, especially with retroarch in them, were taking up to 160 second to build.

Reply:
The timeout doesn't start until after the zip is built. The 30 seconds is just waiting for the browser to start the download.